packages/orchestration/src/exos/local-orchestration-account.js

```diff
@@ -385,39 +421,43 @@ export const prepareLocalOrchestrationAccountKit = (
          * @param {IBCMsgTransferOptions} [opts] if either timeoutHeight or
          *   timeoutTimestamp are not supplied, a default timeoutTimestamp will
          *   be set for 5 minutes in the future
-         * @returns {Promise<void>}
+         * @returns {Vow<void>}
          */
-        async transfer(amount, destination, opts) {
-          trace('Transferring funds from LCA over IBC');
-          // TODO #9211 lookup denom from brand
-          if ('brand' in amount) throw Fail`ERTP Amounts not yet supported`;
+        transfer(amount, destination, opts) {
+          return asVow(() => {
+            trace('Transferring funds from LCA over IBC');
+            // TODO #9211 lookup denom from brand
+            if ('brand' in amount) throw Fail`ERTP Amounts not yet supported`;
 
-          const connectionInfoV = watch(
-            chainHub.getChainInfo('agoric'),
-            this.facets.getChainInfoWatcher,
-            { destination },
-          );
+            const connectionInfoV = watch(
+              chainHub.getChainInfo('agoric'),
+              this.facets.getChainInfoWatcher,
+              { destination },
+            );
 
-          // set a `timeoutTimestamp` if caller does not supply either `timeoutHeight` or `timeoutTimestamp`
-          // TODO #9324 what's a reasonable default? currently 5 minutes
-          // FIXME: do not call `getTimeoutTimestampNS` if `opts.timeoutTimestamp` or `opts.timeoutHeight` is provided
-          const timeoutTimestampV = watch(
-            timestampHelper.getTimeoutTimestampNS(),
-            this.facets.getTimeoutTimestampWatcher,
-            { opts },
-          );
+            // set a `timeoutTimestamp` if caller does not supply either `timeoutHeight` or `timeoutTimestamp`
+            // TODO #9324 what's a reasonable default? currently 5 minutes
+            // FIXME: do not call `getTimeoutTimestampNS` if `opts.timeoutTimestamp` or `opts.timeoutHeight` is provided
+            const timeoutTimestampV = watch(
+              timestampHelper.getTimeoutTimestampNS(),
+              this.facets.getTimeoutTimestampWatcher,
+              { opts },
+            );
 
-          const transferV = watch(
-            allVows([connectionInfoV, timeoutTimestampV]),
-            this.facets.transferWatcher,
-            { opts, amount, destination },
-          );
-          return when(watch(transferV, this.facets.returnVoidWatcher));
+            const transferV = watch(
+              allVows([connectionInfoV, timeoutTimestampV]),
+              this.facets.transferWatcher,
+              { opts, amount, destination },
+            );
+            return watch(transferV, this.facets.returnVoidWatcher);
+          });
         },
-        /** @type {OrchestrationAccount<any>['transferSteps']} */
+        /** @type {PromiseToVow<OrchestrationAccount<any>['transferSteps']>} */
```

Comment:
This usage is a good basis for a type test; that is: in the test, use these names as motivation.